sdk/typescript/src/providers/provider.ts

```diff
@@ -76,6 +78,19 @@ export abstract class Provider {
     pubkey: string
   ): Promise<SuiTransactionResponse>;
 
+  /**
```

Comment:
This is a recently added endpoint on Fullnode(previously `executeTransaction` is only available on Gateway). Since `sui_getNormalizedMoveModulesByPackage` only exists on Fullnode, using `executeTransactionWithRequestType` can remove the need to use both the Gateway and Fullnode